README.md

```diff
@@ -1 +1,191 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- node.js
+- git
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Open your terminal
+
+- Navigate to the directory where you want to clone the project.
+
+- Clone the project using by running the following command:
+
+`git clone https://github.com/boku5250/space-travlers-hub.git`
+
+### Install
+
+Install this project with: node js, run this command `npm install` in the terminal
+
+### Usage
+
+To run the project, execute the following command:
+
+- `npm run build`
+- `npm run start`
+
+### Run tests
+
+To run tests, run the following command:
+
+- `npm run test`
+
+### Deployment
+
+You can deploy this project using: `npm run deploy`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Giorgi Bokuchava**
+
+- GitHub: [@Giorgi Bokuchava](https://github.com/boku52500)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add Rockets]**
+- [ ] **[Add Missions]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project feel free to leave a star ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for motivating me to build this beautiful project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **[How to run this project?]**
+
+  - [You need to have [Node.js](https://nodejs.org/en/download/) in your local machine, and in the root of the project write `npm run start` in the terminal]
+
+- **[Is it open source?]**
+
+  - [Yes you are allowed to use this project]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- [ ] When I click the link to the `MIT.md` file that you added, I get this 404 `error`:

![00Mit](https://user-images.githubusercontent.com/79058364/182822618-fdfb2523-cb23-4060-9f72-75ec8c29e4cd.png)

- [ ] Please update your Readme, it will be nice to add and update this section(`License`) with the type of License and a link to the license file, as this will make your readme more informative and professional. 👍🏻

Reply:
@BenMukebo I have now added the license, sorry to forget about this. It is now available when you click it.